src/layout/msWorkspace/msWindow.js

```diff
@@ -22,6 +22,7 @@ var MsWindow = GObject.registerClass(
                 param_types: [GObject.TYPE_BOOLEAN],
             },
             request_new_meta_window: {},
+            'disconnect-tileable': {},
```

Comment:
This signal name is over specific should be something like "before-kill" or something

Reply:
OK